core/src/main/scala/org/apache/spark/h2o/utils/LogUtil.scala

```diff
@@ -45,9 +50,9 @@ object LogUtil extends org.apache.spark.Logging {
       logWarn()
     } else {
       LogManager.getLogger("water.default").setLevel(Level.toLevel(level))
-      Log._level = levelIdx
+      // FIXME: Log._level = levelIdx
```

Comment:
Why can't this class be in `water.util`? This way we'll need a minor change in h2o.